manifests/a/actualsolution/VolumeLock/2.4/actualsolution.VolumeLock.yaml

```
@@ -0,0 +1,8 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: actualsolution.VolumeLock
+PackageVersion: '2.4'

```

```suggestion
PackageVersion: 2.4.1
```
Please update file path to match this change.